sql/core/benchmarks/WideSchemaBenchmark-results.txt

```diff
@@ -1,117 +1,145 @@
 
+================================================================================================
+many column field read and write
+================================================================================================
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_151-b12 on Mac OS X 10.12.6
+Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
 many column field r/w:                   Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------
-1 cols x 100000 rows (read in-mem)              16 /   18          6.3         158.6       1.0X
-1 cols x 100000 rows (exec in-mem)              17 /   19          6.0         166.7       1.0X
-1 cols x 100000 rows (read parquet)             24 /   26          4.3         235.1       0.7X
-1 cols x 100000 rows (write parquet)            81 /   85          1.2         811.3       0.2X
-100 cols x 1000 rows (read in-mem)              17 /   19          6.0         166.2       1.0X
-100 cols x 1000 rows (exec in-mem)              25 /   27          4.0         249.2       0.6X
-100 cols x 1000 rows (read parquet)             23 /   25          4.4         226.0       0.7X
-100 cols x 1000 rows (write parquet)            83 /   87          1.2         831.0       0.2X
-2500 cols x 40 rows (read in-mem)              132 /  137          0.8        1322.9       0.1X
-2500 cols x 40 rows (exec in-mem)              326 /  330          0.3        3260.6       0.0X
-2500 cols x 40 rows (read parquet)             831 /  839          0.1        8305.8       0.0X
-2500 cols x 40 rows (write parquet)            237 /  245          0.4        2372.6       0.1X
-
-Java HotSpot(TM) 64-Bit Server VM 1.8.0_92-b14 on Mac OS X 10.11.6
-Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz
+1 cols x 100000 rows (read in-mem)              22 /   25          4.6         219.4       1.0X
+1 cols x 100000 rows (exec in-mem)              22 /   28          4.5         223.8       1.0X
+1 cols x 100000 rows (read parquet)             45 /   49          2.2         449.6       0.5X
+1 cols x 100000 rows (write parquet)           204 /  223          0.5        2044.4       0.1X
+100 cols x 1000 rows (read in-mem)              26 /   28          3.9         255.8       0.9X
+100 cols x 1000 rows (exec in-mem)              32 /   35          3.1         319.3       0.7X
+100 cols x 1000 rows (read parquet)             45 /   52          2.2         445.9       0.5X
+100 cols x 1000 rows (write parquet)           275 /  536          0.4        2746.1       0.1X
+2500 cols x 40 rows (read in-mem)              261 /  434          0.4        2607.3       0.1X
+2500 cols x 40 rows (exec in-mem)              624 /  701          0.2        6240.5       0.0X
+2500 cols x 40 rows (read parquet)             196 /  301          0.5        1963.4       0.1X
+2500 cols x 40 rows (write parquet)            687 / 1049          0.1        6870.6       0.0X
```

Comment:
The difference between `best` and `average` is too high in line 32 and line 33.
I'll try to run this on EC2, too.